configs/default.yml

```diff
@@ -2,14 +2,23 @@ datasets:
   - name: JSQuAD
     version: "1.1"
     split: valid
+  - name: MrTyDi
+    version: "1.0"
+    split: test
+    corpus_sample: 100000
+  - name: MIRACL
+    version: "1.0"
+    split: dev
+    sampling_method: "hard_negative"
+    corpus_sample: 200
 #  - name: JSQuAD
 #    version: "1.1"
 #    split: train
 embeddings:
-#  - name: CohereEmbedding
-#    model_name: embed-multilingual-light-v3.0
-#  - name: CohereEmbedding
-#    model_name: embed-multilingual-v3.0
+ - name: CohereEmbedding
+   model_name: embed-multilingual-light-v3.0
+ - name: CohereEmbedding
+   model_name: embed-multilingual-v3.0
   - name: VertexAITextEmbedding
```

Comment:
I tried VertexAITextEmbedding but, with MrTyDi dataset, token length seems too long for VertexAITextEmbedding.
![image](https://github.com/nouu-me/document_vector_search_benchmark/assets/11359334/8c37bf34-92cd-46dc-aa93-3a3fd8f10a94)

How about commenting out all of "cloud embeddings" (Cohere, VertexAI, OpenAI)?